internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -361,7 +386,9 @@ func collectionNameToTableName(name string) string {
 		truncateTo = nameSymbolsLeft
 	}
 
-	return mangled[:truncateTo] + "_" + fmt.Sprintf("%08x", hash32.Sum(nil))
+	i := rand.Intn(100)
+
+	return mangled[:truncateTo] + "_" + fmt.Sprintf("%08x%d", hash32.Sum(nil), i)
```

Comment:
Old metadata references are resolved by adding a random number [0,100) to the hash. Not sure if this is the best approach here.

The probability of a collision is 0.01 which is not great but we could always increase the range or use a timestamp.

We should probably delete / purge old rows periodically but that's outside the scope.

Reply:
Timestamp is more random and I think it's preferable.
But aside from that changing this would change how we generate the table name. Is this safe in terms of compatibility? I see it is only used for table creation so...